pyproject.toml

```diff
@@ -1,29 +1,32 @@
 [tool]
 [tool.poetry]
 name = "mojap-metadata"
-version = "1.2.2"
+version = "1.3.0"
 description = "A python package to manage metadata"
 license = "MIT"
 authors = ["MoJ Data Engineering <dataengineering@digital.justice.gov.uk>"]
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = ">=3.7,<4"
+python = ">=3.7.1,<3.10"
 jsonschema = ">=3.0.0"
 parameterized = "0.7.*"
 PyYAML = ">=5.4.1"
 pyarrow = { version = ">=3.0.0", optional = true }
 etl-manager = { version = "^7.4", optional = true }
+pydbtools = { version = "^4.0.1", optional = true }
 
 [tool.poetry.dev-dependencies]
 flake8 = "^3.8.4"
 black = ">=20.8b1"
 pytest = ">=6.1.2"
 pytest-watch = ">=4.2.0"
+moto = "^2.2.11"
 
 [tool.poetry.extras]
 arrow = ["pyarrow"]
-etl-manager = ["etl-manager"] 
+etl-manager = ["etl-manager"]
+pydbtools = ["pydbtools"]
```

Comment:
```suggestion
glue = ["pydbtools"]
```

Reply:
The extra name is the dependency name then the list is the packages that is dependant on. So the naming convention for the extras above are:
- for the `arrow` converter you need to install `pyarrow`
- for the `etl-manager` converter you need `etl-manager`